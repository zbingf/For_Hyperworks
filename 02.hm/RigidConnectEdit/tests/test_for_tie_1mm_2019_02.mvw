{ safe_quotes_on }
{ MOD_0 = "test_for_tie_1mm_2019_02.h3d" }
{ RES_0 = "test_for_tie_1mm_2019_02.h3d" }
*Id("HyperWorks", "8.*")
*BeginPage()
    *Title("Analysis Results", On)
    *Layout(1)
    *BeginAnimator(Static)
        *CurrentPosition(0)
        *NumberOfSteps(4)
        *Increment(Forward, Frame, 1, BounceOff)
    *EndAnimator()
    *BeginWindow(Animation)
        *BeginGraphic()
            *BeginModel({MOD_0}, "Hyper3D Reader", 1, 1)
                *ResultSource({RES_0}, "Hyper3D Reader", 1, 1)
                *BeginPart(999, "Global", "PART", 999)
                    *Attribute(On, IdOff, 1, Opa, Sha)
                *EndPart()
                *BeginPart(1, "Global", "PART", 1)
                    *Attribute(On, IdOff, 20, Opa, Sha)
                *EndPart()
                *BeginPart(1001, "Orphan-Grids", "PART", 0)
                    *Attribute(Off, IdOff, 37, Opa, Sha)
                *EndPart()
                *BeginGroup("Orphan Nodes", "Off", "Off", "255 255 255", 1, "wire")
                    *BeginSelection(Part, SelectAll, "User_Set", 3)
                        *Add("Orphan-Grids id 1")
                    *EndSelection()
                *EndGroup()
                *BeginMask()
                    *BeginSelection(Part, SelectAll, "User_Set", 0)
                        *Add("Orphan-Grids id 1")
                    *EndSelection()
                *EndMask()
            *EndModel()
            *CurrentModel(0)
            *View(Isometric)
        *EndGraphic()
    *EndWindow()
*EndPage()
