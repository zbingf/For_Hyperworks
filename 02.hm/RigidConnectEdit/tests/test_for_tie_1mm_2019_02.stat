 Program execution started at Sat Oct 01 00:00:29 2022    
	process id = 3392
 Command line arguments:
 D:/github/TclPyHyperWorks/hm/RigidEdit/tests/test_for_tie_1mm_2019_02.fem
 -optskip
 -aif
  
 Expanding dictionary from 0 to 1111

TIME SPENT IN MODULE :    READ;  CPU=       0.08;  WALL=      0.09


START RUNNING MODULE :  PREPR1 ...

TIME SPENT IN MODULE :  PREPR1;  CPU=       0.00;  WALL=      0.00


START RUNNING MODULE :  DESIGN ...

TIME SPENT IN MODULE :  DESIGN;  CPU=       0.00;  WALL=      0.00


START RUNNING MODULE :  PREPRC ...

TIME SPENT IN MODULE :  PREPRC;  CPU=       0.02;  WALL=      0.63


START RUNNING MODULE :  PREPRO ...

   Start running section :  PRDMGR ...

   Time spent in section :  PRDMGR;  CPU=       0.22;  WALL=      0.00


   Start running section :  PRROTD ...

   Time spent in section :  PRROTD;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRP2DR ...

   Time spent in section :  PRP2DR;  CPU=       0.00;  WALL=      0.01


   Start running section :  PRPACO ...

   Time spent in section :  PRPACO;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRPFT1 ...

   Time spent in section :  PRPFT1;  CPU=       0.00;  WALL=      0.00


   Start running section :PRCYSH3D ...

   Time spent in section :PRCYSH3D;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRDMIG ...

   Time spent in section :  PRDMIG;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRPDDM ...

   Time spent in section :  PRPDDM;  CPU=       0.22;  WALL=      0.01


   Start running section :  PRPROF ...

   Time spent in section :  PRPROF;  CPU=       0.00;  WALL=      0.01


   Start running section :  PRPFRF ...

   Time spent in section :  PRPFRF;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRP3DR ...

   Time spent in section :  PRP3DR;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRDPPD ...

   Time spent in section :  PRDPPD;  CPU=       0.00;  WALL=      0.00


   Start running section :PREPOUT0 ...

   Time spent in section :PREPOUT0;  CPU=       0.00;  WALL=      0.00


   Start running section : PREPOUT ...

   Time spent in section : PREPOUT;  CPU=       0.00;  WALL=      0.00


   Start running section :PROUTSET ...

   Time spent in section :PROUTSET;  CPU=       0.00;  WALL=      0.00


   Start running section : ACPVPOP ...

   Time spent in section : ACPVPOP;  CPU=       0.00;  WALL=      0.00


   Start running section :PROUTDYN ...

   Time spent in section :PROUTDYN;  CPU=       0.23;  WALL=      0.00


   Start running section :PROUTDST ...

   Time spent in section :PROUTDST;  CPU=       0.00;  WALL=      0.00


   Start running section :PROUTSTP ...

   Time spent in section :PROUTSTP;  CPU=       0.00;  WALL=      0.00


   Start running section :PROUTEND ...

   Time spent in section :PROUTEND;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRMISD ...

   Time spent in section :  PRMISD;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRP4DR ...

   Time spent in section :  PRP4DR;  CPU=       2.25;  WALL=      0.20


   Start running section :  PROPT1 ...

   Time spent in section :  PROPT1;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRMIS2 ...

   Time spent in section :  PRMIS2;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRPSL4 ...

   Time spent in section :  PRPSL4;  CPU=       0.14;  WALL=      0.15


   Start running section :  PRPMI1 ...

   Time spent in section :  PRPMI1;  CPU=       0.02;  WALL=      0.00


   Start running section :  PRPFT3 ...

   Time spent in section :  PRPFT3;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRPEST ...

   Time spent in section :  PRPEST;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRPMI2 ...

   Time spent in section :  PRPMI2;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRPMMO ...

   Time spent in section :  PRPMMO;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRPMI3 ...

   Time spent in section :  PRPMI3;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRPDSK ...

   Time spent in section :  PRPDSK;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRPSUM ...

   Time spent in section :  PRPSUM;  CPU=       0.00;  WALL=      0.00


   Start running section :  PRPMI4 ...

   Time spent in section :  PRPMI4;  CPU=       0.00;  WALL=      0.00


TIME SPENT IN MODULE :  PREPRO;  CPU=       3.08;  WALL=      0.40


ITERATION   0 (times since start:      1.17     -1.17)

START RUNNING MODULE : CHECKEL ...

TIME SPENT IN MODULE : CHECKEL;  CPU=       0.00;  WALL=      0.01


START RUNNING MODULE :  SOLVER ...

   Start running section :  PRASMB ...

   Time spent in section :  PRASMB;  CPU=       0.00;  WALL=      0.01


   Start running section :  SPASMB ...

   Time spent in section :  SPASMB;  CPU=       0.16;  WALL=      0.15


   Start running section :EIGSLVPR ...

   Solving subcase          1

   Time spent in section :EIGSLVPR;  CPU=       0.00;  WALL=      0.00


   Start running section :AMLSPRP1 ...

   Time spent in section :AMLSPRP1;  CPU=       0.00;  WALL=      0.00


   Start running section :AMLSPRP2 ...

   Time spent in section :AMLSPRP2;  CPU=       0.06;  WALL=      0.08


   Start running section :AMLSPRP3 ...

   Time spent in section :AMLSPRP3;  CPU=       0.02;  WALL=      0.01


   Start running section :AMLSPRP4 ...

   Time spent in section :AMLSPRP4;  CPU=       0.00;  WALL=      0.00


   Start running section :EIGINI_X ...

   Time spent in section :EIGINI_X;  CPU=       0.23;  WALL=      0.23


   Start running section :EIGINP_X ...

   Time spent in section :EIGINP_X;  CPU=       0.03;  WALL=      0.04


   Start running section :SOLEIG_X ...

   Time spent in section :SOLEIG_X;  CPU=       0.80;  WALL=      0.79


   Start running section :EIGRCV_X ...

   Time spent in section :EIGRCV_X;  CPU=       0.00;  WALL=      0.00


   Start running section :  GETEIG ...

   Time spent in section :  GETEIG;  CPU=       0.00;  WALL=      0.00


   Start running section :  OBJFUN ...

   Time spent in section :  OBJFUN;  CPU=       0.00;  WALL=      0.00


TIME SPENT IN MODULE :  SOLVER;  CPU=       1.31;  WALL=      1.32


START RUNNING MODULE :  STRESS ...

TIME SPENT IN MODULE :  STRESS;  CPU=       0.00;  WALL=      0.00


START RUNNING MODULE : ACPOSTP ...

   Start running section :   CMMAP ...

   Time spent in section :   CMMAP;  CPU=       0.00;  WALL=      0.00


   Start running section :   CMCSV ...

   Time spent in section :   CMCSV;  CPU=       0.00;  WALL=      0.00


TIME SPENT IN MODULE : ACPOSTP;  CPU=       0.00;  WALL=      0.00


START RUNNING MODULE : RANDDRV ...

TIME SPENT IN MODULE : RANDDRV;  CPU=       0.00;  WALL=      0.00


START RUNNING MODULE :  RESULT ...

TIME SPENT IN MODULE :  RESULT;  CPU=       0.00;  WALL=      0.00


START RUNNING MODULE :  ERPDRV ...

TIME SPENT IN MODULE :  ERPDRV;  CPU=       0.00;  WALL=      0.00


START RUNNING MODULE :CONVERGE ...

TIME SPENT IN MODULE :CONVERGE;  CPU=       0.00;  WALL=      0.00


START RUNNING MODULE :   PRINT ...

TIME SPENT IN MODULE :   PRINT;  CPU=       0.00;  WALL=      0.00


START RUNNING MODULE : RESULTS ...

   Start running section : RESPREP ...

   Time spent in section : RESPREP;  CPU=       0.00;  WALL=      0.00


   Start running section : RESMEMO ...

   Time spent in section : RESMEMO;  CPU=       0.00;  WALL=      0.00


   Start running section :RESREADY ...

   Time spent in section :RESREADY;  CPU=       0.00;  WALL=      0.00


   Start running section :      HM ...

   Time spent in section :      HM;  CPU=       0.02;  WALL=      0.02


   Start running section :     H3D ...

   Time spent in section :     H3D;  CPU=       0.05;  WALL=      0.06


   Start running section :    MISC ...

   Time spent in section :    MISC;  CPU=       0.00;  WALL=      0.00

  
 ANALYSIS COMPLETED.
  

TIME SPENT IN MODULE : RESULTS;  CPU=       0.06;  WALL=      0.39


START RUNNING MODULE :FINISHUP ...

TIME SPENT IN MODULE :FINISHUP;  CPU=       0.00;  WALL=      0.00

 Detailed disk usage: 
	High usage overall   :    0.0 MiB,
	Solver scratch       :    0.0 MiB,
	Internal database    :    0.0 MiB,
	Out-of-core assembly :    0.0 MiB,
	Temporary files      :    0.0 MiB.

SOLUTION COMPLETED, THE FOLLOWING IS CPU/WALL CLOCK SUMMARY (IN SECONDS):

TOTAL TIME SPENT IN MODULE :READ    ;  CPU=       0.08;  WALL=      0.09
TOTAL TIME SPENT IN MODULE :PREPR1  ;  CPU=       0.00;  WALL=      0.00
TOTAL TIME SPENT IN MODULE :DESIGN  ;  CPU=       0.00;  WALL=      0.00
TOTAL TIME SPENT IN MODULE :PREPRC  ;  CPU=       0.02;  WALL=      0.63
   Total time spent in section :PRDMGR  ;  CPU=       0.22;  WALL=      0.00
   Total time spent in section :PRROTD  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRP2DR  ;  CPU=       0.00;  WALL=      0.01
   Total time spent in section :PRPACO  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRPFT1  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRCYSH3D;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRDMIG  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRPDDM  ;  CPU=       0.22;  WALL=      0.01
   Total time spent in section :PRPROF  ;  CPU=       0.00;  WALL=      0.01
   Total time spent in section :PRPFRF  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRP3DR  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRDPPD  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PREPOUT0;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PREPOUT ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PROUTSET;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :ACPVPOP ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PROUTDYN;  CPU=       0.23;  WALL=      0.00
   Total time spent in section :PROUTDST;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PROUTSTP;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PROUTEND;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRMISD  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRP4DR  ;  CPU=       2.25;  WALL=      0.20
   Total time spent in section :PROPT1  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRMIS2  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRPSL4  ;  CPU=       0.14;  WALL=      0.15
   Total time spent in section :PRPMI1  ;  CPU=       0.02;  WALL=      0.00
   Total time spent in section :PRPFT3  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRPEST  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRPMI2  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRPMMO  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRPMI3  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRPDSK  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRPSUM  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :PRPMI4  ;  CPU=       0.00;  WALL=      0.00
TOTAL TIME SPENT IN MODULE :PREPRO  ;  CPU=       3.08;  WALL=      0.40
TOTAL TIME SPENT IN MODULE :CHECKEL ;  CPU=       0.00;  WALL=      0.01
   Total time spent in section :PRASMB  ;  CPU=       0.00;  WALL=      0.01
   Total time spent in section :SPASMB  ;  CPU=       0.16;  WALL=      0.15
   Total time spent in section :EIGSLVPR;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :AMLSPRP1;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :AMLSPRP2;  CPU=       0.06;  WALL=      0.08
   Total time spent in section :AMLSPRP3;  CPU=       0.02;  WALL=      0.01
   Total time spent in section :AMLSPRP4;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :EIGINI_X;  CPU=       0.23;  WALL=      0.23
   Total time spent in section :EIGINP_X;  CPU=       0.03;  WALL=      0.04
   Total time spent in section :SOLEIG_X;  CPU=       0.80;  WALL=      0.79
   Total time spent in section :EIGRCV_X;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :GETEIG  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :OBJFUN  ;  CPU=       0.00;  WALL=      0.00
TOTAL TIME SPENT IN MODULE :SOLVER  ;  CPU=       1.31;  WALL=      1.32
TOTAL TIME SPENT IN MODULE :STRESS  ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :CMMAP   ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :CMCSV   ;  CPU=       0.00;  WALL=      0.00
TOTAL TIME SPENT IN MODULE :ACPOSTP ;  CPU=       0.00;  WALL=      0.00
TOTAL TIME SPENT IN MODULE :RANDDRV ;  CPU=       0.00;  WALL=      0.00
TOTAL TIME SPENT IN MODULE :RESULT  ;  CPU=       0.00;  WALL=      0.00
TOTAL TIME SPENT IN MODULE :ERPDRV  ;  CPU=       0.00;  WALL=      0.00
TOTAL TIME SPENT IN MODULE :CONVERGE;  CPU=       0.00;  WALL=      0.00
TOTAL TIME SPENT IN MODULE :PRINT   ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :RESPREP ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :RESMEMO ;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :RESREADY;  CPU=       0.00;  WALL=      0.00
   Total time spent in section :HM      ;  CPU=       0.02;  WALL=      0.02
   Total time spent in section :H3D     ;  CPU=       0.05;  WALL=      0.06
   Total time spent in section :MISC    ;  CPU=       0.00;  WALL=      0.00
TOTAL TIME SPENT IN MODULE :RESULTS ;  CPU=       0.06;  WALL=      0.39
TOTAL TIME SPENT IN MODULE :FINISHUP;  CPU=       0.00;  WALL=      0.00

CUMULATIVE RUN TIME:     CPU=        4.55;     WALL=        2.85

