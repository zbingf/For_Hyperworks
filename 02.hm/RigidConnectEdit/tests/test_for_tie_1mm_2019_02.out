 ************************************************************************
 **                                                                    **
 **                                                                    **
 **                    Altair OptiStruct(TM) 2021.1                    **
 **                                                                    **
 **             Advanced Engineering Analysis, Design and              **
 **        Optimization Software from Altair Engineering, Inc.         **
 **                                                                    **
 **   OptiStruct includes an enhanced proprietary version of NASTRAN   **
 **                                                                    **
 **                                                                    **
 **             Windows 10  (Build 9200)  DESKTOP-T8TGPFU              **
 **            16 CPU:  AMD Ryzen 7 3700X 8-Core Processor             **
 **                    24223 MB RAM,  44492 MB swap                    **
 **                                                                    **
 ** Build: 1066780uwn33211_Ce64RB68UH14M:191080-000  46004004000000014 **
 ************************************************************************
 **  COPYRIGHT (C) 1996-2021                 Altair Engineering, Inc.  **
 ** All Rights Reserved.  Copyright notice does not imply publication. **
 **         Contains trade secrets of Altair Engineering, Inc.         **
 ** Decompilation or disassembly of this software strictly prohibited. **
 ************************************************************************
 
  
 *** OptiStruct defaults set from:
   install config file: D:/software/Altair/2021.1/hwsolvers/optistruct.cfg.
  
  
 NOTE # 9199
 MSGLMT=STRICT is active, all messages will be printed.
 You can suppress some less important warning messages by use of
  MSGLMT=BRIEF or UNREF (in config file or in the input data).
  
 NOTE # 1929
 Created RAM scratch area: total size 200 MB.
  
  
 

************************************************************************

OPTIMIZATION FILE AND PARAMETER INFORMATION :
---------------------------------------------

  FEM model file               : 
     D:/github/TclPyHyperWorks/hm/RigidEdit/tests/test_for_tie_1mm_2019_02.fem
  Output files prefix          : 
     D:/github/TclPyHyperWorks/hm/RigidEdit/tests/test_for_tie_1mm_2019_02
  Model read from files:
 (*1)	    D:/github/TclPyHyperWorks/hm/RigidEdit/tests/test_for_tie_1mm_2019_02.fem

************************************************************************

 

************************************************************************

FINITE ELEMENT MODEL DATA INFORMATION :
---------------------------------------

  Total # of Grids (Structural)        :      8741
  Total # of Elements                  :      8410
  Total # of Rigid Elements            :         8
  Total # of Rigid Element Constraints :        48
  Total # of Degrees of Freedom        :     52446
  (Structural)
  Total # of Non-zero Stiffness Terms  :   1403709

 
  Element Type Information
  ------------------------

   CQUAD4   Elements   :     8371     
   CTRIA3   Elements   :       39     

  Load and Boundary Information
  -----------------------------


  Material and Property Information
  ---------------------------------

   PSHELL Cards      :         1
   MAT1   Cards      :         1

************************************************************************

 

************************************************************************

OPTIMIZATION PROBLEM PARAMETERS :
---------------------------------

 NO DESIGN MATERIAL OR DESIGN VARIABLES FOUND  : ANALYSIS ONLY

   ---------------------------
   Frequency Subcase Summary :
   ---------------------------

   ----------   --------  ---------------------  --------
   Subcase ID    SPC ID   Preloading Subcase ID   MODE # 
   ----------   --------  ---------------------  --------
          1         None                               10
   ----------   --------  ---------------------  --------

   Run Type                        :   Analysis Only
   Run at location                 : D:/github/TclPyHyperWorks/hm/RigidEdit/tests/
   Scratch file directory          : D:/github/TclPyHyperWorks/hm/RigidEdit/tests/
                                     Free space:   105.983 GB
   Number of CPU processors        :          1


************************************************************************

 

************************************************************************

MEMORY ESTIMATION INFORMATION :
-------------------------------

 Solver Type is:  Sparse-Matrix Solver
                  Direct Method
                  Block Lanczos Eigenvalue Extraction
 
 Memory (RAM) Allocated in Preprocessor                  :     800 MB
   In addition a ramdisk area was allocated in memory    :     200 MB
 Estimated Memory (RAM) for Out of Core Solution         :      87 MB
 Estimated Memory (RAM) for In-Core Solution             :     186 MB
 
DISK SPACE ESTIMATION INFORMATION :
-----------------------------------

 Estimated Disk Space for Output Data Files              :       6 MB
 Estimated Scratch Disk Space for In-Core Solution       :     417 MB
 Estimated Scratch Disk Space for Out of Core Solution   :     548 MB

************************************************************************


BEGINNING ANALYSIS SOLUTION ....


************************************************************************

ANALYSIS RESULTS :
------------------

  
ITERATION   0
  
 *** INFORMATION # 3454
 MPC, RBE# (and other rigids) and JOINTG constraints for
 subcase 1 will be enforced with elimination of dependent dofs.
  
 (Scratch disk space usage for starting iteration = 54 MB)
(Running in-core solution)

Volume                              =  8.63746E+03   Mass     =  6.78041E-05

                                                  Generalized   Generalized
 Subcase  Mode    Frequency      Eigenvalue        Stiffness        Mass   
       1      1   1.113333E-02   4.893389E-03     4.893389E-03   1.000000E+00
       1      2   1.158299E-02   5.296649E-03     5.296649E-03   1.000000E+00
       1      3   1.241135E-02   6.081323E-03     6.081323E-03   1.000000E+00
       1      4   1.795009E-02   1.272017E-02     1.272017E-02   1.000000E+00
       1      5   1.809838E-02   1.293121E-02     1.293121E-02   1.000000E+00
       1      6   1.891452E-02   1.412376E-02     1.412376E-02   1.000000E+00
       1      7   1.962560E+02   1.520567E+06     1.520567E+06   1.000000E+00
       1      8   3.890180E+02   5.974467E+06     5.974467E+06   1.000000E+00
       1      9   4.734331E+02   8.848648E+06     8.848648E+06   1.000000E+00
       1     10   6.165392E+02   1.500656E+07     1.500656E+07   1.000000E+00


************************************************************************
  
 ANALYSIS COMPLETED.
  

************************************************************************

RESOURCE USAGE INFORMATION
--------------------------

 MAXIMUM MEMORY USED                                       800 MB
   IN ADDITION    200 MB RAMDISK AREA WAS ALLOCATED IN MEMORY
 MAXIMUM DISK SPACE USED                                     1 MB

************************************************************************


************************************************************************

COMPUTE TIME INFORMATION
------------------------

 EXECUTION STARTED                         Sat Oct 01 00:00:29 2022    
 EXECUTION COMPLETED                       Sat Oct 01 00:00:32 2022    
 ELAPSED TIME                                         00:00:02
 CPU TIME                                             00:00:04

************************************************************************

  
                        ***** END OF REPORT *****
  
 For Useful OptiStruct Tips and Tricks, go to the URL:
 http://www.altairhyperworks.com/tips.aspx
